astronomer/providers/amazon/aws/triggers/emr.py

```diff
@@ -64,6 +65,17 @@ async def run(self) -> AsyncIterator["TriggerEvent"]:  # type: ignore[override]
                 else:
                     msg = "EMR Containers sensors completed"
                     yield TriggerEvent({"status": "success", "message": msg})
+
+                if self.max_retries and try_number >= self.max_retries:
```

Comment:
If `max_retries` is an optional param, it has to come as a keyword param with default value as `None` and after positional params in the constructor. https://github.com/astronomer/astronomer-providers/pull/313/files#diff-0a94dcae11767f8098ecf337281ccae74019ebc185787234098d24bd1f34b3e8R29

Reply:
Added the default value